lib/elixir/lib/supervisor.ex

```diff
@@ -529,12 +528,12 @@ defmodule Supervisor do
           | {:max_seconds, pos_integer}
 
   @typedoc "Supported strategies"
-  @type strategy :: :one_for_one | :one_for_all | :rest_for_one
+  @type strategy :: :one_for_one | :one_for_all | :rest_for_one | :simple_one_for_one
 
   # Note we have inlined all types for readability
   @typedoc "The supervisor specification"
   @type child_spec :: %{
-          required(:id) => term(),
+          required(:id) => module() | term(),
```

Comment:
Even though it's redundant it shows what's the default :id

Reply:
I think `atom()` is better as the `module()` is just a consequence of the atom.